app/models/cachingModels.scala

```diff
@@ -17,9 +17,9 @@
 package models
 
 import play.api.libs.json._
-import reactivemongo.bson.BSONObjectID
+import org.bson.types.ObjectId
+import uk.gov.hmrc.mongo.play.json.formats.MongoFormats.Implicits._
 import uk.gov.hmrc.mongo.CreationAndLastModifiedDetail
```

Comment:
CreationAndLastModifiedDetail is part of the simple-reactivemongo distribution. Seems its being pulled in by the mongo-lock library which is not needed by hmrc-mongo. Noticed other repos, create there own version as its just case class with two DateTime fields.